Source/JavaScriptCore/heap/MachineStackMarker.cpp

```diff
@@ -539,7 +539,7 @@ void* MachineThreads::Thread::Registers::stackPointer() const
 #error Unknown Architecture
 #endif
 
-#elif defined(__GLIBC__) && ENABLE(JIT)
+#elif OS(LINUX) && ENABLE(JIT)
```

Comment:
is there a __GLIBC__-like macro being defined for Musl? IMO that should be preferred -- (defined(__GLIBC__) || defined(__MUSL__))